### ISSUE TITLE
How do I add my own custom items?

### ISSUE THREAD

**Issue by Seagate-1TB** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1711]
I am running eclipse on my laptop, and I'm trying to find out how to make custom items (such as fuel rods) with their own stats and everything, but I am new to java coding, how do I do this?

I have git cloned it to a folder called "CUSTOM-HBM-NTM" and I don't know what to do

**Comment by Seagate-1TB** [comment]
I have added the RTG pellets I needed to ModItems.java

**Comment by TheChosenIndividual** [comment]
You should...probably delete this before HBM sees it, ooooh boy is he gonna be pissed with you if he sees it, a nice verbal beatdown

**Comment by BallOfEnergy1** [comment]
why is this created as an issue

**Comment by Vaern** [comment]
getting pissed over... modifying an open-source mod? what are you on about

**Comment by HbmMods** [comment]
For instructions on how to build the mod, consult the README. Items are generally all found and registered in `com.hbm.items.ModItems.java`, create a field at the top, instantiate in the middle and then use `GameRegistry.registerItem` at the bottom.

**Comment by Seagate-1TB** [comment]
So the only thing is to do is create a field, instantiate it, and use GameRegistery.registerItem, I'll start making my images for the RTG pellets

**Comment by Seagate-1TB** [comment]
Right, so How do I make custom discriptions and such that PELLET_RTG requires?
